docs/changelog.md

```diff
@@ -52,6 +50,9 @@ permalink: /changelog/
   engine.clearData(BrowsingData.select(BrowsingData.COOKIES), host = "mozilla.org")
   ```
 
+* **engine-system**:
```

Comment:
Did you add the change to system engine prior to this PR?

Reply:
Yup!